docs/internal/maintenance/ReleaseProcess-release.md

```diff
@@ -8,19 +8,20 @@ Release process - Release
 * [3. Updates to repositories](#3-updates-to-repositories)
 * [4. Determine the new version number](#4-determine-the-new-version-number)
 * [5. Update the Changes file](#5-update-the-changes-file)
-* [6. Set all version numbers](#6-set-all-version-numbers)
-* [7. Update Makefile.PL with required version](#7-update-makefilepl-with-required-version)
-* [8. Create a clean Git working area of *develop branch*](#8-create-a-clean-git-working-area-of-develop-branch)
-* [9. Produce distribution tarballs](#9-produce-distribution-tarballs)
-* [10. Produce distribution zip file](#10-produce-distribution-zip-file)
-* [11. Update Zonemaster repository main _README.md_](#11-update-zonemaster-repository-main-readmemd)
-* [12. Generate documents](#12-generate-documents)
-* [13. Upload to CPAN](#13-upload-to-cpan)
-* [14. Merge develop branch into master](#14-merge-develop-branch-into-master)
-* [15. Create Docker images and upload image to Docker Hub](#15-create-docker-images-and-upload-image-to-docker-hub)
-* [16. Tag the release with git](#16-tag-the-release-with-git)
-* [17. Announce the release](#17-announce-the-release)
-* [18. Merge master into develop](#18-merge-master-into-develop)
+* [6. Set version number](#6-set-version-number)
+* [7. Set version number for GUI](#7-set-version-number-for-gui)
```

Comment:
Suggestion: would it make sense to make the section for GUI versioning a subsection of 6 (Set version number)?

Reply:
Do you think
```
6. Set version number for Perl modules
7. Set version number for GUI
```
is a reasonable solution too?

Reply:
If you really think a new section makes sense (instead of a subsection), then why not.

Reply:
Yes, because you go through those sections the document several times, once per repository. It is easier with focused sections.

Updated as suggested.